code/Pawn/Player.cs

```diff
@@ -1,49 +1,49 @@
 ﻿using Sandbox;
-using TerryForm.Weapons;
+using System.Collections.Generic;
 
 namespace TerryForm.Pawn
 {
-	public partial class Player : Sandbox.Player
+	public partial class Player : Entity
 	{
-		[Net] public Weapon EquippedWeapon { get; set; }
+		public List<Worm> Worms { get; set; }
+		public Worm ActiveWorm { get; set; }
 
-		public override void Respawn()
+		public Player()
 		{
-			SetModel( "models/citizenworm.vmdl" );
+			Worms = new();
 
-			Controller = new WormController();
-			Animator = new WormAnimator();
-			Camera = new Camera();
+			for ( int i = 0; i < 1; i++ )
```

Comment:
It's not difficult to understand what this is for, but right now it would more sense to have a constant/variable for worm counts here rather than just executing this loop once.